stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/avatar/AvatarView.kt

```diff
@@ -18,13 +17,17 @@ public class AvatarView : AppCompatImageView {
     private val borderPaint = Paint(Paint.ANTI_ALIAS_FLAG).apply {
         style = Paint.Style.STROKE
     }
-    private val onlineIndicatorOutlinePaint = Paint().apply {
-        style = Paint.Style.FILL
-        color = Color.WHITE
+    private val onlineIndicatorOutlinePaint by lazy {
+        Paint().apply {
+            style = Paint.Style.FILL
+            color = avatarStyle.onlineIndicatorBorderColor
```

Comment:
What do you think about to move this line inside of the `setStyle()` method?
In the future, if we allow to update the `AvatarStyle` of this view, we won't have a race condition caused because the lazy instance that is not updated after the `AvatarStyle` is updated

Reply:
Done